CONTRIBUTING.md

```diff
@@ -1,5 +1,3 @@
-# Contributing to Acts
-
```

Comment:
Should we keep the header, seeing as this is a standalone markdown file?

Reply:
This breaks the inclusion in the documentation via `.. mdinclude`. The file header leads to a double section title. I can also not remove the section title in the corresponding rst file in docs since sphinx requires a title for each file.

Reply:
Well then you can't include this markdown file in the doc I guess.

Reply:
But I want to. The contribution guide should be available also on the documentation page.

I do not see any issue in removing the header here. Just from the file naming `CONTRIBUTING` it is clear that this contains the contribution guidelines. The links to this file in the Github UI (e.g. at the bottom of the PR page) is also automatically named contribution guidelines. In all cases where the file is used directly, there is no ambiguity as to what it contains.

The other option would be to convert the file to RestructuredText. Then we can include the file verbatim in the documentation.

Reply:
I think there should be a header.

Reply:
Header and no inclusion in the documentation page, header and broken inclusion in the documentation page, or conversion to RestructuredText and header?

Reply:
I guess conversion to rST would be preferrable.

Reply:
Done. I auto-converted the existing file with some minor manual cleanups.